### PR TITLE
feat: change method of reading files

### PR DESCRIPTION
**Pull request by zqq454224016**
feat: change method of reading files

Fix : #89

### CODE REVIEW

File: pkg/repository/chartmuseum.go [hunk 1]
```diff
@@ -134,6 +132,10 @@ func (c *chartmuseum) Poll() {
 	}
 
 	updateRepo := false
```
Comment (Abirdcfly) [inline pkg/repository/chartmuseum.go:134]: https://github.com/kubebb/core/pull/104/files?diff=split#diff-2e9abd09c0abdabe2a30a80ced0000f009dd3e909fc18ad52b113e1189e6affbR187-R191 var `updateRepo` and line 187-191 can also be deleted.

**Comment by bjwswang** [comment]
@zqq454224016 the 2nd commit should be prefixed with `feat/fix/docs...` . pls refer to  https://kubebb.github.io/website/docs/contribute#pr%E8%A7%84%E8%8C%83

**Review by zqq454224016 — COMMENTED** [review]
feat: Update chartmuseum.go

Fix: #89

File: pkg/repository/chartmuseum.go [hunk 2]
```diff
@@ -207,95 +202,17 @@ func (c *chartmuseum) Delete(component *v1alpha1.Component) error {
...
-		}
-
-		if err = yaml.Unmarshal(data, &repositories); err != nil {
-			c.logger.Error(err, "unmarshal response body error")
-			return err
-		}
-		return nil
-	}); err != nil {
-		return nil, fmt.Errorf("timeout, exceeds maximum number of attempts %d", steps)
+	if err = yaml.Unmarshal(data, &repositories); err != nil {
```
Comment (bjwswang) [inline pkg/repository/chartmuseum.go:214]: will unmarshall failed due to `data` is nil?

**Pull request closed without merging**